src/main/kotlin/no/digdir/fdk/statistics/service/StatisticsService.kt

```diff
@@ -0,0 +1,144 @@
+package no.digdir.fdk.statistics.service
+
+import no.digdir.fdk.statistics.model.Concept
+import no.digdir.fdk.statistics.model.DataService
+import no.digdir.fdk.statistics.model.Dataset
+import no.digdir.fdk.statistics.model.Event
+import no.digdir.fdk.statistics.model.InformationModel
+import no.digdir.fdk.statistics.model.Interval
+import no.digdir.fdk.statistics.model.ResourceType
+import no.digdir.fdk.statistics.model.Service
+import no.digdir.fdk.statistics.model.StatisticsObject
+import no.digdir.fdk.statistics.model.TimeSeriesPoint
+import no.digdir.fdk.statistics.model.TimeSeriesRequest
+import no.digdir.fdk.statistics.repository.StatisticsRepository
+import org.springframework.http.HttpStatus
+import org.springframework.stereotype.Component
+import org.springframework.web.server.ResponseStatusException
+import java.time.Duration
+import java.time.LocalDate
+import java.time.ZoneOffset
+import java.time.temporal.ChronoUnit
+
+@Component
+class StatisticsService(private val statisticsRepository: StatisticsRepository) {
+
+    fun storeConceptStatistics(fdkId: String, concept: Concept, timestamp: Long) {
+        statisticsRepository.store(
+            StatisticsObject(
+                id = "$fdkId-$timestamp",
```

Comment:
ikke bedre med uuid?

Reply:
Vil ha med at den aldri lagrer samme ressurs fra samme høsting flere ganger

Hvis vi av en eller annen grunn setter parser-service til å jobbe seg gjennom gamle høstinger så vil den produsere nye events på rdf-parse, med tilfeldig id så kan vi ende opp med den lagrer samme event 2 ganger.

Kan eventuelt lage hash ut fra fdkId+timestamp og bruke det som id, eller unik uuid som id og en UNIQUE-index for de to kolonnene. Et av de to alternativene som blir bedre?

Reply:
eller kanskje bare droppe id og si at primaryKey er kombinasjonen av fdkId og timestamp?